lib/makersbnb.rb

```diff
@@ -0,0 +1,29 @@
+require 'pg'
+
+class MakersBnb_Listings
+
+  def self.create_space(space_name:, price:, description:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'makers_bnb_test')
```

Comment:
we have a setup_database.rb and lib/database_connection.rb which would help to clean up this code. Nothing to change for now but once merged we could make use of these files.